ui/packages/components/src/TriggerDetails/TriggerDetails.tsx

```diff
@@ -124,6 +123,15 @@ export function TriggerDetails({ className, getTrigger, runID }: Props) {
     ];
   }, [trigger]);
 
+  if (error) {
+    throw error;
+  }
```

Comment:
**question**: How do all of these errors get displayed to the user? Within this file and `RunDetails`, each is switched from a toast to just a throw. I'm mostly concerned with the UX and what the user actually experiences. Do you have a screenshot of this? Does the whole section or page fail? Or is there an error boundary that limits the error to only, for example, the Trigger Details panel?

Reply:
The whole page goes into an error state:
![image](https://github.com/user-attachments/assets/f5c0c4ec-36f9-45de-9774-3c733f156557)

There aren't any designs for a more focused error state